Source/WebCore/platform/graphics/texmap/TextureMapperLayer.h

```diff
@@ -242,6 +255,10 @@ class WEBCORE_EXPORT TextureMapperLayer : public CanMakeWeakPtr<TextureMapperLay
 #endif
     bool m_isBackdrop { false };
     bool m_isReplica { false };
+    bool m_recordDamage { false };
```

Comment:
we call `recordDamage` to the method to record a damage, I would call this `m_shouldRecordDamage`

Reply:
Better, I'll use the same `enum class` instead you suggested above.

Reply:
Done, using the newly introduced `enum class DamagePropagation` type.

Reply:
This ended up like this with the suggested change:

```cpp
    Vector<WebCore::IntRect, 1> damageRects;
    if (damage) {
        if (damage->isEmpty())
            damageRects.append({ 0, 0, 0, 0 });
        else
            damageRects = damage->rects();
    }
```

...which is, arguably, easier to understand. Thanks for this suggestion.